packages/data-point-service/examples/express-cache-schema-revalidation.js

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/**
+ *  This example is to show how you can force revalidation via schema validation.
+ *  the idea is to rely on two parts:
+ *  1. adding an outputType reducer using json schema validation.
+ *  2. cache key is created with a unique string hash based on the schema used.
+ *
+ *  This will cause the following behaviors:
+ *  - Every time the schema changes, the cache key changes, this will force a cache revalidation.
```

Comment:
please use a semicolon or period (instead of a comma) after "the cache key changes". it's a bit of a run on sentence now